src/lib/Microsoft.Health.Events/Telemetry/Exceptions/UnauthorizedAccessEventHubException.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Microsoft.Health.Events.Telemetry.Exceptions
 {
-    public sealed class UnauthorizedAccessEventHubException : IomtTelemetryFormattableException
+    public sealed class UnauthorizedAccessEventHubException : CustomerLoggedFormattableException
```

Comment:
Some more classes to derive from `CustomerLoggedFormattableException`:
- [InvalidEventHubExceptions](https://github.com/microsoft/iomt-fhir/blob/main/src/lib/Microsoft.Health.Events/Telemetry/Exceptions/InvalidEventHubException.cs)
[InvalidFhirServiceException](https://github.com/microsoft/iomt-fhir/blob/main/src/lib/Microsoft.Health.Extensions.Fhir.R4/Telemetry/Exceptions/InvalidFhirServiceException.cs)
- [UnauthorizedAccessFhirServiceException](https://github.com/microsoft/iomt-fhir/blob/main/src/lib/Microsoft.Health.Extensions.Fhir.R4/Telemetry/Exceptions/UnauthorizedAccessFhirServiceException.cs)